Makefile

```diff
@@ -30,21 +30,21 @@ pytest:
 	@printf '\n\n*****************\n'
 	@printf '$(color)Running pytest$(off)\n'
 	@printf '*****************\n'
-	pytest tests/flexmock_pytest_test.py
+	pytest tests/test_pytest.py
 
 .PHONY: unittest
 unittest:
 	@printf '\n\n*****************\n'
 	@printf '$(color)Running unittest$(off)\n'
 	@printf '*****************\n'
-	$(PYTHON) -m unittest tests/flexmock_test.py
+	$(PYTHON) -m unittest tests/test_flexmock.py
 
 .PHONY: twisted
 twisted:
 	@printf '\n\n*****************\n'
 	@printf '$(color)Running twisted tests$(off)\n'
 	@printf '*****************\n'
-	$(PYTHON) -c "from twisted.scripts.trial import run; run();" tests/flexmock_pytest_test.py
+	$(PYTHON) -c "from twisted.scripts.trial import run; run();" tests/test_flexmock.py
 
```

Comment:
Should a testtools section be added ?

Reply:
I was thinking that maybe `make test` could just run unittest, doctest and pytest. Then a separate command `make test-integration` could run all the other integrations. CI pipeline will anyway run all the integration tests. What do you think?

Reply:
Yeah I am not sure that we need to put everything under the makefile. I am not sure we need the makefile altogether even :D Let's move the discussion to an issue :)

Reply:
https://github.com/flexmock/flexmock/issues/116